fivetran/connectors/lambda/README.md

```diff
@@ -0,0 +1,101 @@
+# Terraform Module: Fivetran lambda
```

Comment:
Should we add a folder with example lambdas, where we can put e.g. the cloudwatch lambda?